src/ExperimentationService.ts

```diff
@@ -13,7 +12,12 @@ class ExperimentationTelemetry implements IExperimentationTelemetry {
     }
 
     public postEvent(eventName: string, props: Map<string, string>): void {
-        // do nothing
+        const payload: any = { __event_name__: eventName };
+        for (const [key, value] of props) {
```

Comment:
Does `payload = { ...props, ___event_name___: eventName }` work with the `Map<string, string>` type?

Reply:
Seems doesn't, but in `ES2019` we can write it as `payload = { ...Object.fromEntries (props), ___event_name___: eventName }`